recipes/Orange3-Fairness/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "Orange3-Fairness" %}
+{% set version = "0.1.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 18f85525bf1a4b1cf6286a92051b6053ad745826311c037cd40482411b95824b
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python <3.12
+    - setuptools
+  run:
+    - aif360 ==0.5.0
+    - numpy >=1.23.0,<1.24.dev0
+    - orange3 >=3.32.0
+    - python <3.12
+    - scikit-learn =<1.3.2
+    #Not directly a dependancy, but the newer versions are currently having an issue with the scipy dependancy
+
+test:
+  imports:
+    - orangecontrib
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://orangedatamining.com/
+  license: GPL-3.0-only
+  license_family: GPL3
+  license_file: LICENSE
+  summary: "Orange3 add-on for fairness-aware machine learning."
+  doc_url: https://orange3-fairness.readthedocs.io/en/latest/
+  dev_url: https://github.com/biolab/orange3-fairness
+
+extra:
+  recipe-maintainers:
+    - zanmervic
+    - primozgodec
```

Comment:
Looks like this username is all lower case but the original isn't.

Reply:
Does it matter? It works this way in all other recipes I am assigned (https://github.com/conda-forge/orange3-feedstock/blob/main/recipe/meta.yaml).

Do you think it should be changed for aesthetic reasons?

Reply:
The linter is confused. I'm not sure if it matters to GitHub though. Ideally we should be more precise to avoid future changes breaking this.

Reply:
Fixed